cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -183,7 +183,7 @@ func (r *KubeArchiveConfigReconciler) desiredRole(ctx context.Context, kaconfig
 			rules = append(rules, rbacv1.PolicyRule{
 				APIGroups: []string{apiGroup},
 				Resources: []string{strings.ToLower(gvr.Resource.Resource)},
-				Verbs:     []string{"get", "list", "watch"}})
+				Verbs:     []string{"get", "list", "watch", "delete"}})
```

Comment:
This list is (was) alphabetized. Can you put `delete` first?

Reply:
`delete` is now first in the list